### PR TITLE
Fix all sections nav in the footer for non-JS browsers

### PR DESCRIPTION
**Pull request by JustinPinner**
When a non-JS browser visits us, the footer of the page should display the fully-expanded all sections navigation menu, and the menu button at the top of the page should reposition the reader on to the lower section when clicked.

If a JS-enabled browser visits, the all sections menus (at the head and foot of the page) should collapse and expand in response to user interactions.

This PR sets an initial default class for the footer nav section which is replaced with the collapsed class by javascript (which will only ever work if JS is enabled, obvs).

Without JS:
screen shot 2015-08-07 at 17 16 11

With JS:
screen shot 2015-08-07 at 17 16 49

### CODE REVIEW

**Comment by OliverJAsh** [comment]
I thought this used to work? Did we break it somehow?

WRT your frustrations with grunt validation, you can get IDE plugins that show you linting errors inline when editing code. Or we could disable rules if you think we should :smile:

**Comment by JustinPinner** [comment]
Thanks, @OliverJAsh. Alex said it used to work, but I think it may have been broken by the changes for the slim header. Regarding grunt - I just have to get used to its fastidiousness I think, but the IDE plugin may help to preserve my sanity!

File: static/src/javascripts/projects/common/modules/navigation/navigation.js [hunk 1]
```diff
@@ -27,6 +28,10 @@ define([
             }
         },
 
+        jsEnableFooterNav: function () {
+            $('.navigation-container--default').removeClass('navigation-container--default').addClass('navigation-container--collapsed');
```
Comment (sndrs) [inline static/src/javascripts/projects/common/modules/navigation/navigation.js]: this should be in a `fastdom`

Comment (sndrs) [inline static/src/javascripts/projects/common/modules/navigation/navigation.js]: could we not use the `.js-on/off` scope classes to save another dom write?

**Pull request merged**